core/src/main/java/io/questdb/cairo/wal/seq/TableSequencerAPI.java

```diff
@@ -351,6 +367,10 @@ public void resumeTable(TableToken tableToken, long resumeFromTxn) {
         }
     }
 
+    public boolean setApplied(final TableToken tableToken, long writerTxn) {
```

Comment:
Naming is misleading: we have `TableWriter#getAppliedSeqTxn()` method which returns `txWriter.getSeqTxn() + txWriter.getLagTxnCount()` while here we assume `txWriter.getSeqTxn()`. We should have more consistent naming around applied txn (what's visible to table readers) and applied-but-kept-in-lag txn. WDYT?

Reply:
renamed this method to `notifyCommitReadable`